src/zarangollo.md

```diff
@@ -0,0 +1,31 @@
+# Zarangollo
+
+Simple veggie starter/side dish from Murcia (Southeastern Spain)
+
+- ⏲️ Prep time: 5 min
+- 🍳Cook time: 10-15 min
+- 🍽️ Servings: 2
+
+## Ingredients
+
+- Onion
+- Eggplant
+- Olive oil
+- Egg (optional)
```

Comment:
Having explicit amounts here would be useful. I assume from the context in the directions that this calls for 1 onion and 1 eggplant etc, but having explicit values would be ideal I think.